Source/WarpX.cpp

```diff
@@ -853,6 +853,10 @@ WarpX::ReadParameters ()
         pp_warpx.query("do_pml_j_damping", do_pml_j_damping);
         pp_warpx.query("do_pml_in_domain", do_pml_in_domain);
         pp_warpx.query("do_similar_dm_pml", do_similar_dm_pml);
+        v_particle_pml = PhysConst::c;
+        pp_warpx.query("v_particle_pml", v_particle_pml);
+        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(v_particle_pml > 0.7_rt,
```

Comment:
Should this be `0.` instead of `0.7`?
```suggestion
        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(v_particle_pml > 0._rt,
```